frontend/package.json

```
@@ -10,16 +10,16 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^18.2.9",
-    "@angular/common": "^18.2.10",
+    "@angular/animations": "^18.2.11",
+    "@angular/common": "^18.2.11",
     "@angular/compiler": "^18.2.10",
```

Biome

[error] The key @angular/compiler was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)